tests/Query/ExecutorTest.php

```diff
@@ -56,7 +56,11 @@ public function resolveShouldRejectIfRequestIsLargerThan512Bytes()
         Block\await($promise, $this->loop);
     }
 
-    /** @test */
+    /**
+     * @test
+     * @expectedException React\Dns\Query\CancellationException
+     * @expectedExceptionMessage DNS query for igor.io has been cancelled
```

Comment:
Maybe revert moving to annotations since the BC layer for `setExpectedException()` is now available?

Reply:
@jsor Any particular reason why would you want this? Since this layer is temporary, because when this packaging goes to `PHP 7`, we will drop `PHPUnit` 4 and 5 support.

Reply:
In general, `expectException` / `setExpectedException` is preferred over annotations, because exceptions might be thrown before the place where you expected them.

Reply:
What @kelunik said and we should probably polyfill `expectException()`so we can just remove that BC layer when upgrading.

Reply:
I've rolled back the changes. Let's wait for the tests, and we should be ready to go.